lib/import.js

```diff
@@ -113,41 +120,65 @@ class Import {
             if (!options.input) return cb(new Error('options.input required'));
             if (!options.message) return cb(new Error('options.message required'));
 
+            let total = 0; // Total uploaded line delimited features - output in case of fatal error so the user can resume
             let rl;
 
             if (Array.isArray(options.input)) {
                 rl = new ArrayReader(options.input);
-
+                read();
             } else {
-                const filepath = path.resolve(options.input);
-                rl = new readLineSync(filepath);
-
                 // validate the geojson file before uploading starts
-                const geojsonErrs = validateGeojson(filepath, {
-                    ignoreRHR: options.ignoreRHR
-                });
-                if (geojsonErrs.length) return cb(new Error(`Invalid GeoJSON: \n${geojsonErrs}`));
-            }
+                new Schema(self.api).get({}, (err, schema) => {
```

Comment:
Outside of the scope of this PR, but does Schema (and its sibling HecateJs modules) need to be a class if we rarely leverage the reference to the instance? Seems like `api` can just be another param to a bunch of static methods.

Reply:
@mattciferri At the moment the library is designed as both a cli and a library, as a library this makes sense, but for the cli, these sections need to be in a standardized format to allow the API to be build automatically.